c/array-tiling/pnr3_true-unreach-call.i

```diff
@@ -1,10 +1,9 @@
 
-
-int ReadFromPort();
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
 extern void __VERIFIER_assume(int);
 void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
 extern int __VERIFIER_nondet_int(void);
+int ReadFromPort();
```

Comment:
Why do you introduce a declaration and definition separately? Why don't you just move the definition here and not have both? This goes for most fixes.